### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by davaya**
Starting point for revising JADN introductory material to align with Issue #84.  First commit updates abstract.

### CODE REVIEW

File: jadn-v1.1.md [hunk 1]
```diff
@@ -154,46 +159,64 @@ For complete copyright information please see the Notices section in the Appendi
...
+
+The UML specification is organized around the concept of classification, and among its many
+classifiers are DataType and Class. Instances of a DataType are identified only by their value,
+and all instances of a DataType with the same value are considered to be equal instances. DataType
+instances are constants because a different value is by definition a different instance.
+A given value may be classified as an instance of multiple DataTypes, but values can be compared
+for equality only if they are instances of the same type.
+
+Instances of a Class are objects. Objects are not identified by value because two objects
+instantiated from the same Class, even with the same properties, remain distinct: two objects
```
Comment (dlemire60) [inline jadn-v1.1.md]: maybe "... even with the same property _values_ ..." or "... even with the same _values assigned to their_ properties ..."?

File: jadn-v1.1.md [hunk 2]
```diff
@@ -154,46 +159,64 @@ For complete copyright information please see the Notices section in the Appendi
...
-of real-world entities using classes. In contrast, information models model data itself using datatypes.
-A practical distinction is that class models are undirected graphs with an unlimited variety of
-classes and semantic relationships, while information models are directed graphs with a small predefined
-set of base datatypes and only two kinds of relationship: "contain" and "reference".
-Designing an information model from a class/logical model is largely a matter of assigning the kind and
-direction of each relationship, establishing identifiers for all referenceable datatypes, and
-selecting the kind of each datatype from among the base types defined by an information modeling
-language. Converting an information model to a data model means applying serialization rules
-for each base type that produce physical data in the desired format.
+> *An information model is a representation of concepts, relationships, constraints, rules,
```
Comment (dlemire60) [inline jadn-v1.1.md:161]: You've got 3 quotations in here; Y Tina Lee, UML and RDF, but they aren't formatted the same. Suggest normalizing the format as a note (>) in italics w/o quotation marks.

**Review by dlemire60 — APPROVED** [review]
I think this revised introduction does a much more effective job of answering the "why?" question. Couple of minor comments.

**Comment by dlemire60** [comment]
I'd also note that the list of changes in section 1.1 and revision history table in Appendix D should both receive incremental updates as PRs are finalized and merged.

**Comment by davaya** [comment]
I moved the changes from 1.1 to the revisions appendix, Todo: put in table, edit for correctness.

**Comment by dlemire60** [comment]
I've made another pass through the updated PR and I think it looks good. I like the more focused style of the glossary updates.

**Pull request merged**